.circleci/config.yml

```diff
@@ -122,9 +119,7 @@ jobs:
       - *restore_yarn_cache
       - *run_yarn
       - run:
-          environment:
-            RELEASE_CHANNEL: stable
-          command: yarn test-www --maxWorkers=2
+          command: yarn test --project=www --release-channel=stable --ci
```

Comment:
same nit

Edit: I'm just going to stop saying this, but the same nit for all of the
```yaml
run:
  command: ...
```
blocks below :D